### ISSUE TITLE
DOCUMENT_TYPE not used in email templates as it is in vireo3

### ISSUE THREAD

**Issue by smutniak** [TexasDigitalLibrary/Vireo#1580]
src/main/java/org/tdl/vireo/utility/TemplateUtility.java does not have a way to render DOCUMENT_TYPE which exists in vireo3 email templates

**Comment by kaladay** [comment]
This old commit removed document type and may be relevant:
- https://github.com/TexasDigitalLibrary/Vireo/commit/c624093b4741c91d94fed719c153971d7bce5854

**Comment by rmathew1011** [comment]
DOCUMENT_TYPE is referred as `FILE_TYPE` at the front end.

**Comment by cstarcher** [comment]
I do not think this is an issue. I think this is a matter of confusion concerning name refactoring from V3 to V4.

As an example, I will use the Initial Submission email template. This is from V3.

https://github.com/TexasDigitalLibrary/Vireo/blob/3.0.x/conf/emails/SYSTEM_Initial_Submission.email

This is from V4.

https://github.com/TexasDigitalLibrary/Vireo/blob/4.1.3/src/main/resources/emails/SYSTEM_Initial_Submission.email

As you can see, the templates are identical except for the DOCUMENT_TYPE (V3) and the SUBMISSION_TYPE (V4) names. Even the description of the information is the same.

* The document type, typically 'Thesis' or 'Dissertation'

I tested the emails from v4, and the SUBMISSION_TYPE is being populated. Unless I am misunderstanding the OP, I believe this issue should be closed.
